### PR TITLE
Added extra rpath entries to common macOS library install locations.

### PR DESCRIPTION
**Pull request by metasim**
@pomadchin Is it possible for you to publish a SNAPSHOT version so I can easily test the output from CI?

### CODE REVIEW

**Comment by pomadchin** [comment]
Sure, will merge it in once CI is happy

**Comment by metasim** [comment]
Sorry. I thought I had this working locally. Somehow, the `@rpath` substitution is now missing from `libpdalcpp` 😠 

image

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -69,7 +69,8 @@ jobs:
         if: steps.conda-cache.outputs.cache-hit != 'true'
         run: conda install pdal=${{ matrix.pdal }}
 
-      - name: Set LD_LIBRARY_PATH 
+      - name: Set LD_LIBRARY_PATH
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```
Comment (metasim) [inline .github/workflows/ci.yml:73]: More for clarity than anything else.

**Comment by pomadchin** [comment]
ha, what was the change to make it work?

**Comment by metasim** [comment]
> ha, what was the change to make it work?

Basically reverting another change that I thought would eliminate an extra `RC_RPATH` record. I thought I understood the behavior, but it was only passing locally because I relied on `sbt clean` rather than `find . -name target -exec rm -r {} \;`, as the former doesn't clean out all the `cmake` byproducts. See #91

**Comment by metasim** [comment]
How do I find the Maven Central coodinates?

**Comment by pomadchin** [comment]
sonatype snapshots! (https://oss.sonatype.org/content/repositories/snapshots/io/pdal/)

Check the readme, the orange button click! 

image

**Pull request merged**